packages/contracts-bedrock/foundry.toml

```diff
@@ -13,7 +13,6 @@ remappings = [
   'ds-test/=node_modules/ds-test/src'
 ]
 extra_output = ['devdoc', 'userdoc', 'metadata', 'storageLayout']
-bytecode_hash = 'none'
```

Comment:
Not sure how we feel about this, for some reason it was a requirement of echidna. 
Perhaps we can set it up to only disable this setting for echidna runs.